flare/agent_zoo/simple_rl_agents.py

```diff
@@ -62,3 +64,35 @@ def _cts_predict(self, observations, states):
         actions, next_states = self.predict(
             'RL', inputs=dict(sensor=observations), states=dict(state=states))
         return [actions.values()[0][0]], [next_states.values()[0][0]]
+
+
+class ActionNoiseAgent(SimpleRLAgent):
+    """
+    This class extends `SimpleRLAgent` by applying action noise after 
```

Comment:
Add the reason why we generate noise in Agent instead of in Algorithm. Something like "Because each agent needs to maintain its own stochastic process for generating the noise ..."